CONTRIBUTING.md

```diff
@@ -7,10 +7,14 @@ Here is how to do that.
 
 1. Clone this github repo
 2. Make changes
+
```

Comment:
i think that it can do who make a release, so i think that it's not needed here

Reply:
Hi @Arfey 
Could you please take a look CONTRIBUTING.md file one more time? I added "Managing releases in a repository" part in this file. If that is not enough, could you please advise what else should I add?